### PR TITLE
fixed syncing issues

### PR DESCRIPTION
**Pull request by hamdiallam**
introduce a new commitment rate for plasma headers.

Once the commitment rate elapses, we attempt to commit all last plasma headers since the contract's lastCommittedBlock

### CODE REVIEW

File: eth/plasma.go [hunk 1]
```diff
@@ -70,42 +67,67 @@ func InitPlasma(contractAddr common.Address, client Client, finalityBound uint64
...
 
 func (plasma *Plasma) OperatorAddress() (common.Address, error) {
 	return plasma.contract.Operator(nil)
 }
 
-// SubmitBlock proxy. TODO: handle batching with a timmer interrupt
-func (plasma *Plasma) SubmitBlock(block plasmaTypes.Block) error {
-	// only the contract operator can submit blocks
-	if plasma.operatorSession == nil {
+// SubmitBlock proxy. Blocks are committed ever > commitment interval
```
Comment (colin-axner) [inline eth/plasma.go]: comment is a little confusing

File: eth/plasma.go [hunk 2]
```diff
@@ -70,42 +67,67 @@ func InitPlasma(contractAddr common.Address, client Client, finalityBound uint64
...
+		return err
+	}
 
-	_, err := plasma.operatorSession.SubmitBlock(
-		[][32]byte{block.Header},
-		[]*big.Int{big.NewInt(int64(block.TxnCount))},
-		[]*big.Int{block.FeeAmount},
-		plasma.blockNum)
+	firstBlockNum := new(big.Int).Add(lastCommittedBlock, utils.Big1)
+	blockNum := lastCommittedBlock.Add(lastCommittedBlock, utils.Big1)
```
Comment (colin-axner) [inline eth/plasma.go]: Why use lastCommittedBlock and then make blockNum? Couldn't this all work with just blockNum or blockNum/firstBlockNum?

Comment (hamdiallam) [reply eth/plasma.go]: blockNum is incremented while grabbing blocks.

I could subtract the number of  headers from the final block number but that's the same as just holding a reference to the first block numbers.

`blockNum` is actually not a new `big.Int`, its the same reference to `lastCommittedBlock`. `firstBlockNum` creates a new one though.

File: eth/plasma.go (hunk 1 above)
Comment (hamdiallam) [reply eth/plasma.go]: I'll elaborate on it

**Pull request merged**